docs/how_to/search_multiple_wavelengths.rst

```diff
@@ -1,9 +1,12 @@
 .. _how-to-search-for-multiple-wavelengths-with-fido:
```

Comment:
Leave all of the changes to this file out of this PR. We can reformat all of the how-to guides once we finish moving everything.

Reply:
I don't see the harm in these changes.

Reply:
I see now that this is consistent with the formatting of the topic guides added here. I've already implemented most of these formatting fixes in the how-to migration PR: https://github.com/sunpy/sunpy/pull/6948/files#diff-22ef7577fc50d2a8ddb1f13df20688493e2928fcd29c396121c1c362e0bff47d. Could you just add the title formatting change as a suggestion on that PR?